experimental/LieAlgebras/src/AbstractLieAlgebra.jl

```diff
@@ -371,6 +371,9 @@ function lie_algebra(R::Field, S::Symbol, n::Int; cached::Bool=true)
 
   # start temporary workaround # TODO: reenable code above
   type = only(root_system_type(rs))
+  if type == (:F, 4)
+    @warn "GAP uses a non-canonical order of simple roots for F4. Be prepared for unexpected results."
```

Comment:
This warning should be removed:
- We do not do warnings in general.
- Why is it relevant what GAP does? This is OSCAR.

If there is some important information that the user should know about some choices that are different to something else (what exactly?), it should be noted in the docstring.

Reply:
It is in particular annoying for functions that trigger the warning indirectly. It is just unactionable noise, which in most cases is irrelevant.

What is the "near future"?

Also, "GAP" should be replaced by "OSCAR". Otherwise it is not clear what the relevance of GAP is or why one should care.

Reply:
"near future" = next months

OK, I'll then move this warning to the docstring and reformulate it in a way the message is clearer.

Reply:
After talking to @thofma on slack, we decided that throwing a "Not implemented" is the way forward here (for now at least).